audformat/core/table.py

```diff
@@ -540,6 +540,11 @@ def load(
     ):
         r"""Load table data from disk.
 
+        Tables can be stored as PKL and/or CSV files to disk.
+        It will load from the PKL file
+        if available and its modification date is newer,
+        otherwise the CSV file is loaded.
```

Comment:
Should we raise a warning if the modification date of the CSV file is newer than that of the PKL file?

Reply:
Most likely this means the two are not in sync, right?

Reply:
yes, but I would not raise a warning then, because it might be totally on purpose, e.g. you modified the CSV file.

Reply:
yes, but if I do that I should either update the PKL or delete it

Reply:
or you think with this approach we are safe enough that it can never fire back if CSV and PKL are not in sync?

Reply:
This was the whole idea of this approach, to make it more safe.

Another solution would be to raise an error an ask the user to first delete the PKL file.

Reply:
Yes, that would be the safest solution in my opinion.

Reply:
I changed it. It now raises an error if the CSV file is newer.